### PR TITLE
ldm connector switch

### PR DESCRIPTION
**Pull request by djuarezgf**
ldm connector switch
google code style

### CODE REVIEW

**Comment by djuarezgf** [comment]
The ldm connector switch is an adapter that allows using at the same time several ldm's (local data management system). Currently is implemented only for the Query-QueryResult - language but it should be extended to CQL - Measure Reports.
It adapts the original query to every ldm (QueryBuilder) and it collects the results of the ldm's and builds a unique result (ResultsBuilder).

**Review by TasDeniz — APPROVED** [review]

File: src/main/java/de/samply/share/client/quality/report/file/excel/instances/basic/BasicExcelRowElements.java [hunk 1]
```diff
@@ -13,9 +13,9 @@
   private final List<BasicExcelColumnMetaInfo> metaInfos;
 
   /**
-   * Todo.
+   * Contains information (title and eventually link) of each cell of an excel row.
```
Comment (alexanderkiel) [inline src/main/java/de/samply/share/client/quality/report/file/excel/instances/basic/BasicExcelRowElements.java:16]: This comment is more suitable for the Class JavaDoc. On a constructor the comment should start with "Creates a ...".

**Review by TasDeniz — CHANGES_REQUESTED** [review]
Please fix the Google Style Code issues

File: src/main/java/de/samply/share/client/quality/report/file/excel/instances/basic/BasicExcelRowElements.java (hunk 1 above)
Comment (djuarezgf) [reply src/main/java/de/samply/share/client/quality/report/file/excel/instances/basic/BasicExcelRowElements.java:16]: My descriptions of the constructors provide information about the class. But thanks for your advice. I will modify the java docs of the constructors.

**Review by TasDeniz — APPROVED** [review]

**Pull request merged**